src/parser/parser.ts

```diff
@@ -5120,7 +5120,7 @@ export function parseImportMeta(
 
   nextToken(parser, context);
 
-  if (parser.tokenRaw !== 'meta') {
+  if (parser.nodeFlags & (NodeFlags.ExtendedUnicodeEscape | NodeFlags.UnicodeEscape)) {
```

Comment:
@aladdin-add Do you see what I did here? We can now use 'nodeFlags' bitwise masks to figure out if it's an escaped keyword. Gain perf :)
However. We need to pass the "NodeFlag" from scanner to the CST node so this can be checked like this with public API:

'node.flags & (NodeFlags.ExtendedUnicodeEscape | NodeFlags.UnicodeEscape)'